Sources/BraintreePayPalNativeCheckout/BTPayPalNativeTokenizationClient.swift

```diff
@@ -0,0 +1,41 @@
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+#if canImport(BraintreePayPal)
+import BraintreePayPal
+#endif
+
+import PayPalCheckout
+
+class BTPayPalNativeTokenizationClient {
+
+    private let apiClient: BTAPIClient
+
+    init(apiClient: BTAPIClient) {
+        self.apiClient = apiClient
+    }
+
+    func tokenize(request: BTPayPalRequest,
```

Comment:
nit: can we newline everything in the parenthesis?

Reply:
yep, done!